pkgs/development/python-modules/pytest-timeout/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, pexpect
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-timeout";
+  version = "1.2.1";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1kdp6qbh5v1168l99rba5yfzvy05gmzkmkhldgp36p9xcdjd5dv8";
+  };
+  buildInputs = [ pytest ];
```

Comment:
Not propagated?

Reply:
I took `pytest-xdist`, `pytest-datafiles  as reference, none of them seems to use pytest as propagated build input

Reply:
Correct. `pytest` plugins do not propagate `pytest` because it should be possible to use a different `pytest` with a certain plugin.